### PR TITLE
feat: add UI part of student public feedback

### PR DESCRIPTION
**Pull request by sakenalimbayev**

### CODE REVIEW

File: client/src/pages/feedback.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          Submit
+        </Button>
+      </Form>
+    </PageLayoutSimple>
+  );
+}
+
+function getInitialValues(user: User | null) {
+  return {
+    userId: user ? user.id : undefined,
```
Comment (DimaRomaniuk) [inline client/src/pages/feedback.tsx:68]:
```suggestion
    userId: user?.id,
```

File: client/src/services/user.ts [hunk 2]
```diff
@@ -56,6 +56,13 @@ export class UserService {
     });
   }
 
+  async submitStudentFeedback(data: { toUserId: number; comment: string }, courseId: string) {
+    await this.axios.post(`api/course/${courseId}/student/feedback`, {
+      toUserId: Number(data.toUserId),
```
Comment (DimaRomaniuk) [inline client/src/services/user.ts:61]:
```suggestion
      toUserId: data.toUserId,
```

File: client/src/pages/feedback.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import { useMemo, useState } from 'react';
+import { UserService } from 'services/user';
+
+type Props = { router: NextRouter; session: Session };
+type User = { githubId: string; id: number; name: string };
+
+function Page(props: Props) {
+  const { router } = props;
+  const githubId = router.query ? (router.query.githubId as string) : null;
+  const userId = router.query ? (router.query.userId as string) : null;
```
Comment (DimaRomaniuk) [inline client/src/pages/feedback.tsx:14]:
```suggestion
  const githubId = router.query?.githubId ?? null;
  const userId = router.query?.userId ?? null;
```

**Review by apalchys — APPROVED** [review]
feel free to merge when you apply Dima's suggestions

**Pull request merged**